### PR TITLE
moved script to the commonscripts packs

### PR DESCRIPTION
**Pull request by ronykoz**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/26435

### CODE REVIEW

**Comment by yaakovi** [comment]
@ronykoz build is failing

**Comment by ronykoz** [comment]
@yaakovi i'm aware, @dantavori is checking that out

**Comment by ronykoz** [comment]
@amshamah419 could you please go over the RN?

File: Packs/CommonScripts/ReleaseNotes/1_2_4.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+### Scripts
+##### CalculateTimeDifference
+  - Moved the script CalculateTimeDifference to this pack.
```
Comment (amshamah419) [inline Packs/CommonScripts/ReleaseNotes/1_2_4.md:4]:
```suggestion
  - Moved the script `CalculateTimeDifference` to the `CommonScripts` pack.
```

**Comment by ronykoz** [comment]
Force merging due to unrelated problems

**Pull request merged**